products/appengine/api.yaml

```
@@ -440,3 +440,37 @@ objects:
             name: 'shell'
             description: |
               The format should be a shell command that can be fed to bash -c.
+  - !ruby/object:Api::Resource
+    name: 'ApplicationUrlDispatchRules'
+    description: |
+      Rules to match an HTTP request and dispatch that request to a service.
+    base_url: ''
```

I'm near-confident you can set this value, which will set the `project` field on the resource automatically.

```suggestion
    base_url: 'apps/{{project}}'
```

---

I dont know, lete me try to test with this.

---

tested and its working, I will add this change.